gradle/libs.versions.toml

```diff
@@ -0,0 +1,49 @@
+[versions]
+agp = "8.5.2"
+junit = "4.12"
+gson = "2.9.0"
+msgpack = "0.8.11"
+java-websocket = "1.5.3"
+vcdiff = "0.1.1"
+hamcrest = "1.3"
+nanohttpd = "2.3.0"
+mockito = "1.10.19"
+concurrentunit = "0.4.2"
+slf4j = "1.7.30"
+build-config = "5.4.0"
+firebase-messaging = "22.0.0"
+android-test = "0.5"
```

Comment:
Have we added firebase as a new dependency? Couldn't find it in old dependencies

Reply:
Okay, seems this is a Android specific dependency. Just wanted to make sure this doesn't get added into ably-java jar